src/ontology/uberon-edit.obo

```diff
@@ -190087,6 +190087,24 @@ is_a: UBERON:0001145 {source="FMA"} ! ovarian vein
 intersection_of: UBERON:0001145 ! ovarian vein
 intersection_of: drains UBERON:0002119 ! left ovary
 
+[Term]
+id: UBERON:0700019
+name: parallel fiber
+def: "A parallel fiber results from the axon of a cerebellar granule cell ascending through the molecular layer where it synapses on the dendritic branches of Purkinje cells and stellate cells." [GO:1990032] {comment="doi:10.1007/978-94-007-1333-8_63"}
+comment: This a grouping class for parallel fibers in any species. If annotating a parallel fiber in any species other than teleost fish, consider using 'parallel fiber, bifurcated'.
+is_a: UBERON:0006134 ! nerve fiber
```

Comment:
After a cursory review, I found that they are in the cerebellum. Consider adding 'part of' some cerebellum https://doi.org/10.3389/fncir.2020.611841